### PR TITLE
Add the chat page route in index.js

### LINKED ISSUE #12: Add the chat page route in index.js
The chat template should be rendered at /chat.

### PR DESCRIPTION
**Pull request by therohitramesh**
Fixes #12

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -10,5 +10,9 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/chat', (req, res) => {
+	res.render('chat');
```
Comment (sumedh123) [inline index.js]: Indentation.

Comment (sumedh123) [reply index.js]: Make it 2 spaces please.

Comment (therohitramesh) [reply index.js]: Ok alright.

Comment (therohitramesh) [reply index.js]: @sumedh123 I've made the changes you had requested. Please check.

**Comment by sumedh123** [comment]
@therohitramesh Good work.

**Pull request merged**